### ISSUE TITLE
Installing strong-mesh-client for strongloop dependency. 

### ISSUE THREAD

**Issue by luncht1me** [strongloop/strong-mesh-client#48]
Trying to npm install strong-mesh-client; fails on minifier.js;;;

```
> strong-mesh-client@2.0.2 install C:\Users\Guldan\AppData\Roaming\npm\node_modules\strong-mesh-client
> node proxy/build-client.js

 [DEBUG] Here is where Browserify put your modules:
 [DEBUG] Here is how I'm mapping your code:
C:\Users\Guldan\AppData\Roaming\npm\node_modules\strong-mesh-client\node_modules\minifyify\lib\minifier.js:353
        throw new Error('File with no mappings: ' + file)
        ^

Error: File with no mappings: node_modules\strong-mesh-models\index.js
    at C:\Users\Guldan\AppData\Roaming\npm\node_modules\strong-mesh-client\node_modules\minifyify\lib\minifier.js:353:15
    at C:\Users\Guldan\AppData\Roaming\npm\node_modules\strong-mesh-client\node_modules\minifyify\lib\minifier.js:413:5
    at C:\Users\Guldan\AppData\Roaming\npm\node_modules\strong-mesh-client\node_modules\lodash._baseeach\index.js:110:11
    at baseForOwn (C:\Users\Guldan\AppData\Roaming\npm\node_modules\strong-mesh-client\node_modules\lodash._baseeach\index.js:52:10)
    at C:\Users\Guldan\AppData\Roaming\npm\node_modules\strong-mesh-client\node_modules\lodash._baseeach\index.js:80:14
    at Object.each (C:\Users\Guldan\AppData\Roaming\npm\node_modules\strong-mesh-client\node_modules\lodash.foreach\index.js:26:9)
    at Minifier.eachSource (C:\Users\Guldan\AppData\Roaming\npm\node_modules\strong-mesh-client\node_modules\minifyify\lib\minifier.js:412:5)
    at Minifier.transformMap (C:\Users\Guldan\AppData\Roaming\npm\node_modules\strong-mesh-client\node_modules\minifyify\lib\minifier.js:349:8)
    at C:\Users\Guldan\AppData\Roaming\npm\node_modules\strong-mesh-client\node_modules\minifyify\lib\minifier.js:245:25
    at ConcatStream.<anonymous> (C:\Users\Guldan\AppData\Roaming\npm\node_modules\strong-mesh-client\node_modules\concat-stream\index.js:36:43)
C:\Users\Guldan\AppData\Roaming\npm
`-- (empty)

npm ERR! Windows_NT 10.0.10586
npm ERR! argv "c:\\Program Files\\nodejs\\node.exe" "C:\\Users\\Guldan\\AppData\\Roaming\\npm\\node_modules\\npm\\bin\\npm-cli.js" "install" "strong-mesh-client" "--msvs_version=2015" "--python=C:\\python27\\python.exe" "--global"
npm ERR! node v6.2.2
npm ERR! npm  v3.7.3
npm ERR! code ELIFECYCLE

npm ERR! strong-mesh-client@2.0.2 install: `node proxy/build-client.js`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the strong-mesh-client@2.0.2 install script 'node proxy/build-client.js'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the strong-mesh-client package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node proxy/build-client.js
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs strong-mesh-client
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls strong-mesh-client
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     C:\Users\Guldan\Documents\GitHub\Select-Real-Estate\npm-debug.log
npm ERR! code 1
```

////

Windows 10 64bit, Python 2.7, MSVS 2015;

**Comment by rmg** [comment]
@luncht1me what is the reason for using `strong-mesh-client@2.0.2`?

**Comment by rmg** [comment]
@luncht1me from the issue title I'm going to answer my own question with a guess: you installed `strong-mesh-client@2.0.2` (or more likely `strong-mesh-client@latest`) in an effort to debug the issue you were seeing with `npm install -g strongloop`.

Thank you very much for your efforts! This should have been resolved with #58 and `strong-mesh-client@1.6.5`.

Please re-open if you are still having this problem.
